util/testutil/integration/sandbox.go

```diff
@@ -59,6 +59,8 @@ func (sb *sandbox) NewRegistry() (string, error) {
 
 func (sb *sandbox) Cmd(args ...string) *exec.Cmd {
 	if len(args) == 1 {
+		// \\ being stripped off for Windows paths, convert to unix style
```

Comment:
make sure to rebase with `moby/moby:master` as this was merged in https://github.com/moby/buildkit/commit/13fefceb212d8be81e8382a2cb35db89cc5b53c1